tests/test_app.py

```diff
@@ -0,0 +1,118 @@
+import os
+from unittest.mock import Mock, patch 
+
+import pytest
+from dotenv import find_dotenv, load_dotenv
+
+from todo_app import app
+
+TEST_BOARD_ID = os.environ.get('BOARD_ID')
```

Comment:
Here we specify this _before_ we load the `.env.test` file - we should instead move this to where it's used, e.g. inside `mock_get_cards`